### DISCUSSION TITLE
What is the EOL and Support policy for KEDA releases

### DISCUSSION THREAD

**Discussion by mooneeb** [kedacore/keda#4949] [Q&A / Need Help] [answered]
Hello!

I just wanted to enquire about the EOL and Support policy for KEDA releases. I looked through the documentation and although i was able to find information about the [compatibility of KEDA releases with K8s](https://keda.sh/docs/2.11/operate/cluster/#kubernetes-compatibility), I wasn't able to figure out what the support policy is defined by the maintainers for the KEDA releases themselves. 

Looking at the Github releases, it seems like KEDA follows `n` and `n-1` support policy (because KEDA `v2.8.2` was released in between `v2.9.1` and `v2.9.2`), but I'd appreciate it if someone could help provide clarity on this matter or point me in the direction of the right documentation to go through.

Thank you so much!

**Comment by tomkerkhove** [comment, marked as answer]
Hi,

Do these help?

- https://keda.sh/support/
- https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md
- https://github.com/kedacore/governance/blob/main/RELEASES.md
- https://github.com/kedacore/governance/blob/main/SUPPORT.md

**Reply by mooneeb** [reply]
@tomkerkhove Thank you! These links were very helpful. Just wanted to confirm one thing just to ensure my understanding is correct going through the [Deprecations](https://github.com/kedacore/governance/blob/main/DEPRECATIONS.md) doc, it's written that 

> The KEDA runtime, which consists of the operator and metric server, will provide support on the deprecated features for 6 months (currently the equivalent of 2 releases) after the deprecation was announced.

So is it safe to say KEDA will support upto `n-1` releases. For eg, right now, `v2.11.x` and `v2.10.x` will be considered as supported. and once `v2.12` is released, then `v2.11.x` and `v2.12.x` will be the recommended go-to KEDA versions?

**Reply by zroubalik** [reply]
Yes, please bear in mind that the support we are talking about is just a community (best effort) support.

**Reply by mooneeb** [reply]
Yup, understood! Thank you so much!

**Comment by shazib-summar** [comment]
@tomkerkhove @zroubalik thank you so much for the clarification but I had the same question and was lucky to find this discussion thread. Should we add this to GitHub readme, or the [keda.sh docs](https://github.com/kedacore/keda-docs/blob/main/content/docs/2.11/deploy.md) for more visibility? The support policy is something that should be easily discoverable so the user knows which version to use.

**Reply by tomkerkhove** [reply]
This is already on our website though:
![image](https://github.com/kedacore/keda/assets/4345663/59229261-3198-4005-89fc-cb4c20ce1a67)

But based on what I'm hearing, it might be better to move it to top-level navigation instead then?

**Reply by tomkerkhove** [reply]
Same for breaking changes and deprecations :)

**Reply by zroubalik** [reply]
I would keep the breaking changes as they are, to not have the top level menu too long. But moving there support makes sense, if it is gonna help with visibility.

**Reply by mooneeb** [reply]
If i may give my 2 cents, but just a reference link to the supported versions url in the README could be helpful and it would definitely increase visibility.
